README.md

```diff
@@ -70,6 +84,12 @@ collections:
     version: 3.1.1
 ```
 
+or using the ansibble-galaxy command as follows
```

Comment:
```suggestion
A specific version of amazon.aws collection can be installed using the ansible-galaxy command as follows
```

Reply:
The line - "A specific version of amazon.aws collection can be installed" , is already mentioned previously. I will fix typo.